app/pages/docs/what-is-nextjs.mdx

```diff
@@ -1,14 +1,14 @@
 ---
-title: What is Next.js?
-sidebar_label: What is Next.js?
+title: Wat is Next.js?
+sidebar_label: Wat is Next.js?
 ---
 
-Next.js is baked into Blitz and almost all features of Next.js are supported. For more info on the differences see here: [Why use Blitz instead of Next.js](https://blitzjs.com/docs/why-blitz)
+Next.js is ingebakken in Blitz en bijna alle functies van Next.js worden ondersteund. Voor meer info over de verschillen zie hier: [Waarom Blitz gebruiken in plaats van Next.js](https://blitzjs.com/docs/why-blitz)
 
-[**Next.js**](https://nextjs.org/) is a popular React Framework with many built-in features, such as:
+[**Next.js**](https://nextjs.org/) is een populair React Framework met veel ingebouwde functies, zoals:
 
-* Pre-rendering, both static generation \(SSG\) and server-rendering \(SSR\) are supported on a per-page basis
-* Automatic code splitting for faster page loads
-* Client-side routing with optimized prefetching
-* Development environment which supports Hot Module Replacement
-* Fully extendable
+* Pre-rendering, zowel static generation \(SSG\) als server-rendering \(SSR\) worden ondersteund op een per-pagina basis
```

Comment:
static generation -> statische generatie? Kan voor mij wel allebei hoor.
_per-pagina basis_ vind ik wat raar klinken. Misschien eerder iets als: _op paginaniveau_ of zo?